src/main/java/liquibase/ext/databricks/database/DatabricksConnection.java

```diff
@@ -144,7 +143,9 @@ public int getDatabaseMinorVersion() throws DatabaseException {
     /////////////////////////////////////////////////// copy from parent ///////////////////////////////////////////////////
     @Override
     protected String getConnectionUrl() throws SQLException {
-        return con.getMetaData().getURL();
+        String rawUrl = con.getMetaData().getURL();
+        String updatedUrl = rawUrl + "UserAgentEntry=Liquibase;EnableArrow=0";
```

Comment:
Could there be cases where you need to add a `;` at the beginning of this appended string - like if the user doesn't add this?

Reply:
Yea probably. Ill change this and make it check first. I think I should check for the before and after ";"

Reply:
Made this fix. Let me know what you think.